devicetypes/smartthings/zwave-water-valve.src/zwave-water-valve.groovy

```diff
@@ -20,11 +20,11 @@ metadata {
 		capability "Refresh"
 		capability "Sensor"
 
-		fingerprint deviceId: "0x1006", inClusters: "0x25"
-		fingerprint mfr: "0173", prod: "0003", model: "0002", deviceJoinName: "Leak Intelligence Leak Gopher Water Shutoff Valve"
-		fingerprint mfr: "021F", prod: "0003", model: "0002", deviceJoinName: "Dome Water Main Shut-off"
-		fingerprint mfr: "0157", prod: "0003", model: "0002", deviceJoinName: "EcoNet Bulldog Valve Robot"
-		fingerprint mfr: "0152", prod: "0003", model: "0512", deviceJoinName: "POPP Secure Flow Stop"
+		fingerprint deviceId: "0x1006", inClusters: "0x25", deviceJoinName: "Valve"
+		fingerprint mfr: "0173", prod: "0003", model: "0002", deviceJoinName: "Leak Gopher Valve" //Leak Intelligence Leak Gopher Water Shutoff Valve
```

Comment:
I looked it up and the brand is "Leak Intelligence". The Gopher is the model.

Reply:
"The Gopher" is more like a line of products, and I would say that this 'name' is more highlighted, than a "Leak Intelligence". (Something like AOne and Aurora)

Reply:
So should I go back to 'Leak Intelligence'? It's the last one we've got doubts about

Reply:
Eh. It's fine. You've got conflicts now because of me, though.

Reply:
Resolved